data_psi.go

```diff
@@ -24,6 +24,28 @@ const (
 	PSITableTypeUnknown = "Unknown"
 )
 
+type PSITableTypeID uint16
```

Comment:
Why the prefix `TableTypeID` instead of `TableID`?

This is valid for all `TableTypeID` in this page

Reply:
good call, will change